TerminalDocs/customize-settings/global-settings.md

```diff
@@ -299,22 +299,6 @@ This determines the word delimiters used in a double-click selection. Word delim
 
 ___
 
```

Comment:
```suggestion

## Scroll speed

This is the number of rows to scroll at a time with the mouse wheel. This will override the system setting if the value is not zero or `"system"`.

**Property name:** `rowsToScroll`

**Necessity:** Optional

**Accepts:** Integer

**Default value:** `"system"

<br />

> [!IMPORTANT]
> The `"rowsToScroll"` setting is no longer available in [Windows Terminal Preview](https://aka.ms/terminal-preview/), version 1.2. This version of Windows Terminal will always use the value configured in the system Mouse settings panel.

___

```